lottie/src/main/java/com/airbnb/lottie/model/content/LBlendMode.java

```diff
@@ -0,0 +1,78 @@
+package com.airbnb.lottie.model.content;
+
+import android.os.Build;
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+import androidx.core.graphics.BlendModeCompat;
+
+/**
+ * Lottie BlendMode,
+ * not to be confused with Paint.BlendMode in android graphics core,
+ * which we will rely on for rendering.
+ */
+public enum LBlendMode {
+  NORMAL,
+  MULTIPLY,
+  SCREEN,
+  OVERLAY,
+  DARKEN,
+  LIGHTEN,
+  COLOR_DODGE,
+  COLOR_BURN,
+  HARD_LIGHT,
+  SOFT_LIGHT,
+  DIFFERENCE,
+  EXCLUSION,
+  HUE,
+  SATURATION,
+  COLOR,
+  LUMINOSITY,
+  ADD,
+  HARD_MIX;
+
+  @Nullable
+  @RequiresApi(api = Build.VERSION_CODES.Q)
+  public BlendModeCompat toNativeBlendMode() {
+    switch (this){
+      case NORMAL:
+        return null; // BlendMode.CLEAR?
+      case MULTIPLY:
+        return BlendModeCompat.MULTIPLY;
+      case SCREEN:
+        return BlendModeCompat.SCREEN;
+      case OVERLAY:
+        return BlendModeCompat.OVERLAY;
+      case DARKEN:
+        return BlendModeCompat.DARKEN;
+      case LIGHTEN:
+        return BlendModeCompat.LIGHTEN;
+      case COLOR_DODGE:
+        return BlendModeCompat.COLOR_DODGE;
+      case COLOR_BURN:
+        return BlendModeCompat.COLOR_BURN;
+      case HARD_LIGHT:
+        return BlendModeCompat.HARD_LIGHT;
+      case SOFT_LIGHT:
+        return BlendModeCompat.SOFT_LIGHT;
+      case DIFFERENCE:
+        return BlendModeCompat.DIFFERENCE;
+      case EXCLUSION:
+        return BlendModeCompat.EXCLUSION;
+      case HUE:
+        return BlendModeCompat.HUE;
+      case SATURATION:
+        return BlendModeCompat.SATURATION;
+      case COLOR:
+        return BlendModeCompat.COLOR;
+      case LUMINOSITY:
+        return BlendModeCompat.LUMINOSITY;
+      case ADD:
+        return BlendModeCompat.PLUS;
+      case HARD_MIX:
+        return null;  // todo: fix this
```

Comment:
Is there an explicit action item here or is it "wait for platform support"?

Reply:
I think it would be "wait for platform support" at this time. There doesn't seem to be an equivalent to had_mix at this time if we're implementing it like this.

Reply:
Let's make the comment reflect that then.